src/components/TertiaryButton/TertiaryButton.styles.js

```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components';
+
+export const TertiaryButtonWrapper = styled.button`
+  padding: var(--smallest-space) var(--medium-space);
+
+  text-align: center;
+
+  border: 0.1rem solid;
+  border-color: var(--lightblue);
```

Comment:
remove the `border-color` property and move `var(--light-blue)` up to the `border` property
i.e.
`border: 0.1rem solid var(--lightblue);`